packages/builder/src/components/backend/DataTable/buttons/ExistingRelationshipButton.svelte

```diff
@@ -22,13 +27,18 @@
     })
   }
 
-  const afterSave = ({ action }) => {
+  const afterSave = async ({ action }) => {
```

Comment:
This doesn't need to be async, but it does no harm either.

Reply:
woops, don't know why I made it async!